app/assets/stylesheets/searchworks4/masthead.css

```diff
@@ -10,3 +10,11 @@
     --bs-link-color-rgb: var(--stanford-cardinal-rgb);
   }
 }
+
+.navbar {
+  .badge {
+    --bs-badge-color: var(--stanford-black);
+
+    background-color: var(--stanford-fog-light);
```

Comment:
why do we set background color if we're already applying `.bg-fog-light`?

Reply:
Ah because bg-fog-light isn't a class that does anything. That is cruft. I thought it was a class but it wasn't.